app/src/main/java/de/danoeh/antennapod/ui/screen/playback/SleepTimerDialog.java

```diff
@@ -169,6 +272,34 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return builder.create();
     }
 
+    private void refreshExtendButtons() {
+        final SleepTimeConfig selectedConfig = allSleepConfig.get(SleepTimerPreferences.getSleepTimerType());
+
+        int counter = 0;
+        for (Button button : List.of(
+                extendSleepFiveMinutesButton,
+                extendSleepTenMinutesButton,
+                extendSleepTwentyMinutesButton)) {
+            final SleepEntryConfig entryConfig = Objects.requireNonNull(selectedConfig).sleepEntries.get(counter++);
+            // button text resource can be either string or plural string
+            try {
+                button.setText(getString(selectedConfig.buttonTextResourceId, entryConfig.displayValue));
+            } catch (Resources.NotFoundException ex) {
+                button.setText(getContext().getResources()
```

Comment:
Exceptions as normal control flow are a bit unusual. I think the easier option would be to just set the text explicitly

Reply:
Not sure what you mean here, I've extracted the `setText` functionality to a separate method.

Reply:
I wanted to change the string for the minutes to a plural string as well, and even made the required changes, but reverted because all of the translations for this string will become invalid (for little to no gain).